composer.json

```diff
@@ -29,16 +29,24 @@
         "php": ">=7.1.0",
         "ext-bcmath": "*",
         "ext-gmp": "*",
-        "guzzlehttp/guzzle": "~6.0",
         "psr/container": "^1.0",
-        "kornrunner/keccak": "^1.0"
+        "kornrunner/keccak": "^1.0",
+        "psr/http-message": "^1.0",
+        "php-http/client-implementation": "^1.0",
+        "php-http/async-client-implementation": "^1.0",
+        "php-http/httplug": "^1.0",
+        "php-http/message-factory": "^1.0",
+        "php-http/discovery": "^1.0"
     },
     "require-dev": {
         "twig/twig": "^2.0",
         "phpunit/phpunit": "^6.3",
         "iotaledger/kerl": "dev-master as 1.0",
         "nikic/php-parser": "^3.1",
-        "phpdocumentor/reflection-docblock": "^4.2"
+        "phpdocumentor/reflection-docblock": "^4.2",
+        "php-http/mock-client": "^1.0",
+        "php-http/message": "^1.0",
+        "guzzlehttp/psr7": "^1.0"
```

Comment:
why do you need guzzle in test environment ?

Reply:
I followed the implementation guide at http://docs.php-http.org/en/latest/httplug/library-developers.html - I guess the mock does not include a functionality to create responses / requests.

Reply:
You don't need to create an instance of it. Take a look at this: http://docs.php-http.org/en/latest/clients/mock-client.html
You should mock the request or response. Maybe with prophezie?

Reply:
Adjusted.